src/Microsoft.ML.StandardLearners/Standard/MultiClass/MetaMulticlassTrainer.cs

```diff
@@ -54,7 +55,7 @@ public abstract class ArgumentsBase
 
         public TrainerInfo Info { get; }
 
-        public TScalarTrainer PredictorType;
+        public Func<TScalarTrainer> ScalarTrainerFunc;
```

Comment:
>ScalarTrainerFunc [](start = 36, length = 17)

As Pete mentioned earlier, we actually don't need this, and we can use the same TScalarTrainer to train multiple models. So we can get rid of this, and also from the GetTrainer() method, instantiate _trainer from the args in the ctor if singleEstimator is null, and then just use _trainer to fit all the models.

Reply:
I approve conditional on reverting this change

---
In reply to: [232054155](https://github.com/dotnet/machinelearning/pull/1574#discussion_r232054155) [](ancestors = 232054155)